README.md

```diff
@@ -1,19 +1,76 @@
 # Kashmir [![NPM version][npm-image]][npm-url]
```

Comment:
`kashmir` is taken on npm. 
@swarad07 @taher-pro what alternatives do you suggest?
Riffing off of the JS community we could have *create-drupal-theme*